tests/events/client/test_events_client.py

```diff
@@ -51,6 +53,20 @@ def test_errors_when_missing_api_url_and_ephemeral_disabled():
             get_events_client()
 
 
+async def test_prefect_api_tls_insecure_skip_verify_setting_set_to_true(
+    monkeypatch, ephemeral_settings
```

Comment:
nit: can we type hint these? been trying to do this for new tests

Reply:
done